cogs/events.py

```diff
@@ -32,11 +33,66 @@ async def on_error(self, event, *args, **kwargs):
             title=event
         )
 
-        if not self.dev_mode:
-            await self.error_hook.send(embed=em)
+        if not constants.DEV_MODE:
+            # await self.error_hook.send(embed=em)
+            await self.client.send_message(embed=em)
         else:
             traceback.print_exc()
 
+    @commands.Cog.listener()
+    async def on_guild_join(self, guild: discord.Guild):
+
+        hw = self.client.get_user(constants.hotwired_id)
```

Comment:
I believe there is a way to get bot id directly. You should take a look at [`discord.Bot.user`](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html?highlight=bot#discord.ext.commands.Bot.user) it seems like it's a reference to bot which means you could simply do `self.client.user.id` instead of saving it in constants.
This ID is also mentioned on line 63

Reply:
i will check it asap